src/index.js

```diff
@@ -25,23 +29,44 @@ const TRIM_PR_NAME_LENGHT = 35;
  * The default cron expression described as:
  * At minute 0 past hour 9 and 18 on every day-of-week from Monday through Friday.
  */
-const TO_DO_TIME = process.env.TO_DO_TIME || '0 9,20 * * 1-5';
+const TO_DO_TIME = config.to_do_time || '0 9,20 * * 1-5';
 /**
  * The default cron expression described as:
  * At minute 0 past hour 9 and 18 on every day-of-week from Monday through Friday.
  */
-const PR_TIME = process.env.PR_TIME || '0 9,20 * * 1-5';
+const PR_TIME = config.pr_time || '0 9,20 * * 1-5';
 /**
  * The default cron expression described as:
  * At 21:00 on every day-of-week from Monday through Friday.
  */
-const MEETING_TIME = process.env.MEETING_TIME || '0 21 * * 1-5';
+const MEETING_TIME = config.meeting_time || '0 21 * * 1-5';
 const octokit = new Octokit({ auth: TOKEN });
 
 const MEMBERS_QUERY = require('./queries/members');
 const CARDS_QUERY = require('./queries/cards');
 const ISSUE_QUERY = require('./queries/issue');
 const PR_QUERY = require('./queries/pr');
+const { Console } = require('console');
+
+/**
+ * Parse mention list to string
+ * and build mention map that links github to telegram usernames.
+ */
+function parse_mention(mentions) {
```

Comment:
Can we move valued/utility code fragments to separate files? The `index.js` should be just an entry point.

Reply:
Thank you. I'm gonna think about it.

Reply:
you can start with moving this method to the utility file